fix: remove unapproved images & add logo license info

karlbaumhauer opened the pull request:
## Description:
- removes all un-approvd images from placeholder package
- adds information on logos used in our components package

## Definition of Reviewable:
- [x] Documentation is created/updated
- [x] Stories (features, a11y) are created/updated

mariohamann reviewed (changes requested):
Fortunately this is not a Breaking Change!

Maybe the message didn't reach you, but placeholders were published 3 months ago the last time. As the images were added in a "docs" merge, no release was triggered so far.

Therefore we can savely make a "chore" commit, as this will neither impact current versions of components nor placeholders.

One note in addition: Are you sure, that none of the images is part of a screenshot test? At least for the final deployment, please remove the [skip chromatic] flag, to ensure fresh tests.

karlbaumhauer commented:
> Fortunately this is not a Breaking Change!
> 
> Maybe the message didn't reach you, but placeholders were published 3 months ago the last time. As the images were added in a "docs" merge, no release was triggered so far.
> 
> Therefore we can savely make a "chore" commit, as this will neither impact current versions of components nor placeholders.
> 
> One note in addition: Are you sure, that none of the images are part of a screenshot test? At least for the final deployment, please remove the [skip chromatic] flag, to ensure fresh tests.

@mariohamann / @smfonseca I replaced the sds-logo with the ui-brand-mark and added this image to storybook assets instead of the placeholder package.

I also removed the [skip chromatic] to make sure its not f❌ng on any template.

@MartaPintoTeixeira / @coraliefeil pls have a look and you might need to change the Figma template as well (from what I saw there).
